dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -62,6 +65,15 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
     getBindings();
     getChartVersion(chartID, chartVersion);
     getChartValues(chartID, chartVersion);
+
+    if (hr) {
+      this.setState({
+        appValues: hr.spec.values,
+        namespace: hr.metadata.namespace,
+        releaseName: hr.metadata.name,
+        valuesModified: true,
+      });
```

Comment:
if a release object is provided to the deployment form then the component state is initialized with parameters from the release object.